rules/impl.bzl

```diff
@@ -14,6 +14,11 @@ load(
     _ANDROID_LINT_TOOLCHAIN_TYPE = "ANDROID_LINT_TOOLCHAIN_TYPE",
     _utils = "utils",
 )
+load(
+    "@rules_android//providers:providers.bzl",
+    "AndroidLibraryResourceClassJarProvider",
+    "AndroidLibraryAarInfo"
+)
 
```

Comment:
Can you also add `JavaInfo` loads as well? They can be loaded from `rules_java` now.

Reply:
Is like this okay? e75150a5434f0fb9fe26d430c5018504672787d3

Reply:
That looks correct!